lib/CGameState.cpp

```diff
@@ -3541,3 +3554,48 @@ CRandomGenerator & CGameState::getRandomGenerator()
 	//logGlobal->traceStream() << "Fetching CGameState::rand with seed " << rand.nextInt();
 	return rand;
 }
+
+bool CPathfinder::addTeleportTwoWay(const CGTeleport * obj) const
+{
+	if(allowTeleportTwoWay)
+	{
+		if(ETeleportChannelType::BIDIRECTIONAL == gs->getTeleportChannelType(obj->channel, hero->tempOwner))
+			return true;
+	}
+	return false;
+}
```

Comment:
Is this one like really needed? I don't like this kind of formatting as long lines are harder to read.